### ISSUE TITLE
[GoogleMapsOverlay] Tooltip never appears on Mobile browser

### ISSUE THREAD

**Issue by hughsheehy** [visgl/deck.gl#5201]
#### Description
I have a scatterplot using GoogleMapOverlay. Works beautifully on desktop and on Chrome emulation of a mobile phone. Also displays and pans/zooms wonderfully on Chrome on Android. However, the tooltips hardly ever render on Chrome on Android. Well, it rendered once. But that was the only time. 

Love the framework by the way!  Been playing with it for a while. 

#### Repro Steps
View this site on mobile. https://icy-flower-0dfee1e03.azurestaticapps.net/  Tooltips have never worked (for me) on mobile.  Works fine on desktop. (takes a few secs to load...there's a big datafile)

The code is at https://jsfiddle.net/wotapalaver/xg3tu4ef/

The dataset is a BIG GeoJson. It's linked straight in the fiddle.  
There's lots of this. 
******************************************
{"type": "FeatureCollection", "features": [{"type": "Feature", "geometry": {"type": "Point", "coordinates": [-2.096, 57.142]}, "properties": {"Name": "Union Square Car Park", "Address": "Union Square", "Town": "Aberdeen", "Country": "United Kingdom", "Continent": "EU", "Access Comments": "10x 32amp Type 2 chargers with 10 bays on level 1 of multistorey car park, signposted with plug symbol. Charging availability Mon to Sat 09:00 to 23:00, Sun 11.00 to 23.00. ", "Operator": {"ID": 19, "Title": "Zero Carbon World", "WebsiteURL": "http://zerocarbonworld.org/", "AddressInfo": "null_entry", "ContactEmail": "info@zerocarbonworld.com"}, "Usage Cost": "Free but Parking charges apply", "Connectors": [{"ID": 151, "ConnectionTypeID": 25, "ConnectionType": {"ID": 25, "Title": "Type 2 (Socket Only)", "FormalName": "IEC 62196-2 Type 2", "IsDiscontinued": false, "IsObsolete": false}, "Reference": "null_entry", "StatusTypeID": 50, "StatusType": {"ID": 50, "Title": "Operational", "IsOperational": true, "IsUserSelectable": true}, "LevelID": 2, "Level": {"ID": 2, "Title": "Level 2 : Medium (Over 2kW)", "Comments": "Over 2 kW, usually non-domestic socket type", "IsFastChargeCapable": false}, "Amps": 32, "Voltage": 230, "PowerKW": 7.0, "CurrentTypeID": 10, "CurrentType": {"ID": 10, "Title": "AC (Single-Phase)", "Description": "Alternating Current - Single Phase"}, "Quantity": 10, "Comments": null}, {"ID": 180674, "ConnectionTypeID": 3, "ConnectionType": {"ID": 3, "Title": "BS1363 3 Pin 13 Amp", "FormalName": "BS1363 / Type G", "IsDiscontinued": "null_entry", "IsObsolete": null}, "Reference": "null_entry", "StatusTypeID": 0, "StatusType": {"ID": 0, "Title": "Unknown", "IsOperational": "null_entry", "IsUserSelectable": true}, "LevelID": 2, "Level": {"ID": 2, "Title": "Level 2 : Medium (Over 2kW)", "Comments": "Over 2 kW, usually non-domestic socket type", "IsFastChargeCapable": false}, "Amps": 13, "Voltage": 230, "PowerKW": 3.0, "CurrentTypeID": 10, "CurrentType": {"ID": 10, "Title": "AC (Single-Phase)", "Description": "Alternating Current - Single Phase"}, "Quantity": 1, "Comments": null}], "UUID": "17946796-C870-4E12-AEE6-CF765A49A91F"}}, {"type": "Feature", "geometry": {"type": "Point", "coordinates": [-5.96830897916788, 54.5718808033914]}, "properties": {"Name": "Hurst Nissan Centre Belfast", "Address": "62 Boucher Road", "Town": "Belfast", "Country": "United Kingdom", "Continent": "EU", "Access Comments": "null_entry", "Operator": {"ID": 7, "Title": "Nissan UK Dealer Network", "WebsiteURL": "http://www.nissan.co.uk", "AddressInfo": "null_entry", "ContactEmail": "null_entry"}, "Usage Cost": "Free", "Connectors": [{"ID": 154, "ConnectionTypeID": 2, "ConnectionType": {"ID": 2, "Title": "CHAdeMO", "FormalName": "IEC 62196-3 Configuration AA", "IsDiscontinued": "null_entry", "IsObsolete": null}, "Reference": "null_entry", "StatusTypeID": 50, "StatusType": {"ID": 50, "Title": "Operational", "IsOperational": true, "IsUserSelectable": true}, "LevelID": 3, "Level": {"ID": 3, "Title": "Level 3:  High (Over 40kW)", "Comments": "40KW and Higher", "IsFastChargeCapable": true}, "Amps": 125, "Voltage": 400, "PowerKW": 50.0, "CurrentTypeID": 20, "CurrentType": {"ID": 20, "Title": "AC (Three-Phase)", "Description": "Alternating Current - Three Phase"}, "Quantity": 1, "Comments": null}, {"ID": 109229, "ConnectionTypeID": 3, "ConnectionType": {"ID": 3, "Title": "BS1363 3 Pin 13 Amp", "FormalName": "BS1363 / Type G", "IsDiscontinued": "null_entry", "IsObsolete": null}, "Reference": "null_entry", "StatusTypeID": 50, "StatusType": {"ID": 50, "Title": "Operational", "IsOperational": true, "IsUserSelectable": true}, "LevelID": 2, "Level": {"ID": 2, "Title": "Level 2 : Medium (Over 2kW)", "Comments": "Over 2 kW, usually non-domestic socket type", "IsFastChargeCapable": false}, "Amps": 13, "Voltage": 230, "PowerKW": 3.0, "CurrentTypeID": 10, "CurrentType": {"ID": 10, "Title": "AC (Single-Phase)", "Description": "Alternating Current - Single Phase"}, "Quantity": 2, "Comments": null}], "UUID": "5FAE63DC-8C96-436F-AAE7-2F124A18AE3D"}}, 
***********************************************************

#### Environment (please complete the following information):
Works on all desktops and mobile emulations on desktop. 
Using Latest Version of the library. 
Doesn't work on Chrome 86 on Android 10

#### Logs
Nothing in console.

**Comment by hughsheehy** [comment]
Confirmed also same issue on iPad.

**Comment by Pessimistress** [comment]
The tooltip feature is tied to Google Maps' `mousemove` event. If it's never fired on mobile then tooltip won't work.

We could respond to the `click` event on mobile if you have any suggestions on how to reliably detect mobile vs desktop.

**Comment by hughsheehy** [comment]
I discovered the "problem"...or rather I found a solution. 

The sample code I used - from Google - did not have any viewport meta tags. 

I installed a tag and immediately the tooltip works. So I expect that the base code(either in Google or in deck.gl), built for desktop, has tap targets that are so small they are somehow incompatible with mobile browsers. That's my guess anyway. 

So this isn't a bug as much as (at most) a clarification for the documentation.
